docs/medical-api/api-reference/fhir/consolidated-data-query-get-single.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Get Single Consolidated Data Query status"
+description: "Get the status of a specific query for patient's consolidated data."
+api: "GET /medical/v1/patient/{id}/consolidated/query/{requestId}"
+---
+
+This endpoint returns the status of the specified query.
+
+To trigger a new query you can
+[start a consolidated query](/medical-api/api-reference/fhir/consolidated-data-query-post). The result of the query will be sent through a [Webhook request](/medical-api/handling-data/webhooks#patient-consolidated-data)
+in FHIR-compliant format.
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+<ParamField path="requestId" type="string" required>
+  The ID of consolidated query request.
+</ParamField>
+
+## Response
+
+Returns the status of the specified query.
+
+<ResponseExample>
+
+```javascript Metriport SDK
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+
+const metriport = new MetriportMedicalApi(apiToken);
+
+const status = await metriport.getSingleConsolidatedQueryStatus(patientId, requestId);
```

Comment:
Can we break the line to minimize horizontal scroll on the docs.

From

![image](https://github.com/user-attachments/assets/024801c4-95b4-4f7f-8a54-547746c0cfa9)

...to

![image](https://github.com/user-attachments/assets/5ec3b164-c33e-4c90-8aa4-a52a0a6a642d)


Also, it would be good to update the SDK tester, it helps to validate the code that goes on the docs is working, both syntactically and actually running the SDK (needs to publish to NPM first) - [here](https://github.com/metriport/metriport/blob/develop/packages/sdks/typescript/tester-node/src/mapi/consolidated-query.ts#L49).

Reply:
addressed